src/komponenter/kalkulator/Topp/Topp.tsx

```diff
@@ -1,115 +1,48 @@
+                        30 uker
+                    </strong>
+                </Normaltekst>
+                <Normaltekst>Kalkulatoren regner ut:</Normaltekst>
+                <ul className="topp__liste">
+                    <Normaltekst tag="li">
+                        Hvor lenge den ansatte har vært permittert
                     </Normaltekst>
-                </div>
-                <div className={'kalkulator__18mnd-linje'}>
-                    <Element className={'kalkulator__18mnd-illustrasjon-tekst'}>
-                        18 måneder
-                    </Element>
-                </div>
-                <Datovelger
-                    tjenesteBestemtFeilmelding={feilMelding}
-                    className={'initial-datovelger'}
-                    overtekst={'Siste dag i perioden'}
-                    onChange={(event) => {
-                        if (datoValidering(event.currentTarget.value)) {
-                            props.set18mndsPeriode(event.currentTarget.value);
-                            props.setEndringAv('datovelger');
-                        }
-                    }}
-                    value={props.sisteDagIPeriode}
-                />
-            </div>
-            <div className={'kalkulator__velg-ny-periode-info'}>
-                <img
-                    alt={'kalender ikon'}
-                    className={'kalkulator__ikon-kalender'}
-                    src={kalender}
-                />
-                <div>
-                    <Normaltekst>
-                        {
-                            'Permitteringsukene beregnes alltid i et tidsrom på 18 måneder av gangen.  Du kan endre 18 måneders perioden ved å bruke datovelgeren over til høyre.'
-                        }
+                    <Normaltekst tag="li">Når du skal betale lønn</Normaltekst>
+                </ul>
+                <Normaltekst>
+                    <strong>Denne informasjonen trenger du</strong>
+                </Normaltekst>
+                <ul className="topp__liste">
+                    <Normaltekst tag="li">
+                        Periodene den ansatte har vært permittert
                     </Normaltekst>
-                    <br />
-                    <Normaltekst>
-                        {`Dagen du har valgt som sluttdato for perioden er ${formaterDato(
-                            props.sisteDagIPeriode
-                        )}. 18 måneders perioden er tidsrommet fra og med ${formaterDato(
-                            finnDato18MndTilbake(props.sisteDagIPeriode)
-                        )} til og med ${formaterDato(
-                            props.sisteDagIPeriode
-                        )} .`}
+                    <Normaltekst tag="li">
+                        Eventuelle fravær under permitteringen
                     </Normaltekst>
-                </div>
+                </ul>
+            </div>
+            <div className="topp__stegindikator">
```

Comment:
Denne fungerer ikke som tiltenkt ennå. La den bare til for å få det visuelle på plass før demo. Muligens vi burde revurdere bruken, den fungerer ikke helt som jeg trodde.